migrations/2024-02-24-034523_replaceable-schema/down.sql

```diff
@@ -0,0 +1,1009 @@
+    FOR EACH ROW
+    EXECUTE FUNCTION comment_aggregates_comment ();
+
+CREATE TRIGGER comment_aggregates_score
+    AFTER INSERT OR DELETE ON comment_like
+    FOR EACH ROW
+    EXECUTE FUNCTION comment_aggregates_score ();
+
+CREATE TRIGGER community_aggregates_comment_count
+    AFTER INSERT OR DELETE OR UPDATE OF removed,
+    deleted ON comment
+    FOR EACH ROW
+    EXECUTE FUNCTION community_aggregates_comment_count ();
+
+CREATE TRIGGER community_aggregates_community
+    AFTER INSERT OR DELETE ON community
+    FOR EACH ROW
+    EXECUTE FUNCTION community_aggregates_community ();
+
+CREATE TRIGGER community_aggregates_post_count
+    AFTER DELETE OR UPDATE OF removed,
+    deleted ON post
+    FOR EACH ROW
+    EXECUTE FUNCTION community_aggregates_post_count ();
+
+CREATE TRIGGER community_aggregates_post_count_insert
+    AFTER INSERT ON post REFERENCING NEW TABLE AS new_post
+    FOR EACH STATEMENT
+    EXECUTE FUNCTION community_aggregates_post_count_insert ();
+
+CREATE TRIGGER community_aggregates_subscriber_count
+    AFTER INSERT OR DELETE ON community_follower
+    FOR EACH ROW
+    EXECUTE FUNCTION community_aggregates_subscriber_count ();
+
+CREATE TRIGGER delete_follow_before_person
+    BEFORE DELETE ON person
+    FOR EACH ROW
+    EXECUTE FUNCTION delete_follow_before_person ();
+
+CREATE TRIGGER person_aggregates_comment_count
+    AFTER INSERT OR DELETE OR UPDATE OF removed,
+    deleted ON comment
+    FOR EACH ROW
+    EXECUTE FUNCTION person_aggregates_comment_count ();
+
+CREATE TRIGGER person_aggregates_comment_score
+    AFTER INSERT OR DELETE ON comment_like
+    FOR EACH ROW
+    EXECUTE FUNCTION person_aggregates_comment_score ();
+
+CREATE TRIGGER person_aggregates_person
+    AFTER INSERT OR DELETE ON person
+    FOR EACH ROW
+    EXECUTE FUNCTION person_aggregates_person ();
+
+CREATE TRIGGER person_aggregates_post_count
+    AFTER DELETE OR UPDATE OF removed,
+    deleted ON post
+    FOR EACH ROW
+    EXECUTE FUNCTION person_aggregates_post_count ();
+
+CREATE TRIGGER person_aggregates_post_insert
+    AFTER INSERT ON post REFERENCING NEW TABLE AS new_post
+    FOR EACH STATEMENT
+    EXECUTE FUNCTION person_aggregates_post_insert ();
+
+CREATE TRIGGER person_aggregates_post_score
+    AFTER INSERT OR DELETE ON post_like
+    FOR EACH ROW
+    EXECUTE FUNCTION person_aggregates_post_score ();
+
+CREATE TRIGGER post_aggregates_comment_count
+    AFTER INSERT OR DELETE OR UPDATE OF removed,
+    deleted ON comment
+    FOR EACH ROW
+    EXECUTE FUNCTION post_aggregates_comment_count ();
+
+CREATE TRIGGER post_aggregates_featured_community
+    AFTER UPDATE ON post
+    FOR EACH ROW
+    WHEN ((old.featured_community IS DISTINCT FROM new.featured_community))
+    EXECUTE FUNCTION post_aggregates_featured_community ();
+
+CREATE TRIGGER post_aggregates_featured_local
+    AFTER UPDATE ON post
+    FOR EACH ROW
+    WHEN ((old.featured_local IS DISTINCT FROM new.featured_local))
```

Comment:
All of these trigger conditions seem to be missing from the new ones you created.

Reply:
Fixed using a filter in the update statement